### ISSUE TITLE
Error on install: Array and string offset access syntax with curly braces is deprecated

### ISSUE THREAD

**Issue by aladrach** [cloudgrayau/pitch#3]
When attempting to install the plugin, I see the following error, and I'm unable to proceed.

PHP 7.4.11
Craft Pro 3.5.14

```
yii\base\ErrorException: Array and string offset access syntax with curly braces is deprecated in C:\Users\adam\Repositories\testsite\vendor\cloudgrayau\pitch\src\Pitch.php:108
Stack trace:
#0 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\web\ErrorHandler.php(76): yii\base\ErrorHandler->handleError(8192, 'Array and strin...', 'C:\\Users\\adam\\R...', 108)
#1 C:\Users\adam\Repositories\testsite\vendor\composer\ClassLoader.php(444): craft\web\ErrorHandler->handleError(8192, 'Array and strin...', 'C:\\Users\\adam\\R...', 108, Array)
#2 C:\Users\adam\Repositories\testsite\vendor\composer\ClassLoader.php(444): include()
#3 C:\Users\adam\Repositories\testsite\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\Users\\adam\\R...')
#4 [internal function]: Composer\Autoload\ClassLoader->loadClass('cloudgrayau\\pit...')
#5 [internal function]: spl_autoload_call('cloudgrayau\\pit...')
#6 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\services\Plugins.php(878): is_subclass_of('cloudgrayau\\pit...', 'craft\\base\\Plug...')
#7 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\services\Plugins.php(478): craft\services\Plugins->createPlugin('pitch')
#8 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\controllers\PluginsController.php(49): craft\services\Plugins->installPlugin('pitch', NULL)
#9 [internal function]: craft\controllers\PluginsController->actionInstallPlugin()
#10 C:\Users\adam\Repositories\testsite\vendor\yiisoft\yii2\base\InlineAction.php(57): call_user_func_array(Array, Array)
#11 C:\Users\adam\Repositories\testsite\vendor\yiisoft\yii2\base\Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#12 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\web\Controller.php(190): yii\base\Controller->runAction('install-plugin', Array)
#13 C:\Users\adam\Repositories\testsite\vendor\yiisoft\yii2\base\Module.php(528): craft\web\Controller->runAction('install-plugin', Array)
#14 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\web\Application.php(274): yii\base\Module->runAction('plugins/install...', Array)
#15 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\web\Application.php(577): craft\web\Application->runAction('plugins/install...', Array)
#16 C:\Users\adam\Repositories\testsite\vendor\craftcms\cms\src\web\Application.php(253): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#17 C:\Users\adam\Repositories\testsite\vendor\yiisoft\yii2\base\Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#18 C:\Users\adam\Repositories\testsite\web\index.php(22): yii\base\Application->run()
#19 {main}
```

**Comment by aladrach** [comment]
Update: appears to be a PHP 7.4 incompatibility issue, works on 7.3, 7.2.

**Comment by cloudgrayau** [comment]
Thanks, that has now been fixed with 1.0.4.
